src/emitter.js

```diff
@@ -214,13 +214,17 @@ CKEDITOR.define( [ 'eventinfo', 'utils' ], function( EventInfo, utils ) {
 		 * @param {...*} [args] Additional arguments to be passed to the callbacks.
 		 */
 		fire: function( event, args ) {
-			var callbacks = getCallbacksIfAny( this, event );
+			var eventInfo = event instanceof EventInfo && event;
```

Comment:
Aren't the changes in this file and `src/eventinfo.js` a leftover? If not, then they miss tests and changes in documentation.